sycl/doc/design/CompileTimeProperties.md

```diff
@@ -111,32 +111,32 @@ template </* ... */> class
 } // namespace sycl::ext::oneapi
 ```
 
-The device compiler only uses the
-`[[__sycl_detail__::add_ir_attributes_global_variable()]]` attribute when the
-decorated type is used to create an [LLVM IR global variable][3] and the global
-variable's type is either:
+When the device compiler creates an [LLVM IR global variable][3] of the 
+annotated class it will use the
+`[[__sycl_detail__::add_ir_attributes_global_variable()]]` attribute to generate
```

Comment:
I think we will need to change the name. You are no longer adding IR attributes.

Reply:
This is another vote for @steffenlarsen idea. Noted.

Reply:
I have separated this into a different intrinsic.